appium/tests/specs/mock/setup/CheckRevokedKeyUpdate.spec.ts

```diff
@@ -48,6 +48,17 @@ describe('SETUP: ', () => {
       // Because we already have revoked key on local
       await NewMessageScreen.setAddRecipient(contactEmail);
       await NewMessageScreen.checkAddedRecipientColor(contactName, 0, 'red');
+
+      // Now check if other recipients display fine 
+      const testEmail = 'test@example.com';
+      mockApi.attesterConfig = {
+        servedPubkeys: {
+          [testEmail]: ekmKeySamples.key1.pub!
+        }
+      };
+      await NewMessageScreen.deleteAddedRecipient(0);
+      await NewMessageScreen.setAddRecipient(testEmail);
+      await NewMessageScreen.checkAddedRecipientColor('Test2', 0, 'green');
```

Comment:
Not another recipient. Another functioning key (with different fingerprint than the revoked one) for the same recipient as before.

Reply:
You were indeed correct. 👍 
It broke functionality you mentioned and I fixed that issue.